frontend/src/SelectQuestionPage.js

```diff
@@ -1,12 +1,13 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useContext } from 'react';
 import { useNavigate } from "react-router-dom";
 import axios from 'axios';
+import { LanguageContext } from "./Contexts/Contexts";
 
 const SelectQuestionPage = () => {
     const [questions, setQuestions] = useState([]);
     const [questionID, setQuestionID] = useState();
     let navigate = useNavigate();
-
+    const { language} = useContext(LanguageContext);
```

Comment:
Is the `useContext` and `LanguageContext` needed here?
If not, I would remove the unused imports.